.github/workflows/generate-v3.yml

```diff
@@ -0,0 +1,46 @@
+name: Generate V3 Samples
+
+on:
+  schedule:
+    - cron: '0 */1 * * *'
+  repository_dispatch:
+    types: [ generate-v3 ]
+
+jobs:
+  generate-v3:
+    runs-on: ubuntu-latest
+    timeout-minutes: 45
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: Install NDK
```

Comment:
Just for my own knowledge - what is the NDK used for?

Reply:
The NDK appears to be required by the `stripDebugDebugSymbols` task as part of the `./gradlew build` process. I added this step since it caused the following error:

> No version of NDK matched the requested version 20.0.5594570. Versions available locally: 21.0.6113669

Associated build failure: https://github.com/googlemaps/android-samples/runs/665615681?check_suite_focus=true